test/GPv2Settlement/OrderRefunds.t.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.8;
+
+import {Helper} from "./Helper.sol";
+import {Order} from "test/libraries/Order.sol";
+
+enum FreeFunctionVariant {
+    FreeFilledAmountStorage,
+    FreePreSignatureStorage
+}
+
+abstract contract Variant is Helper {
+    FreeFunctionVariant internal freeFn;
```

Comment:
Nit suggestion of making this immutable, for the reader's peace of mind that this will never change in a test.

```suggestion
    FreeFunctionVariant internal immutable freeFn;
```